util/customTransforrmationsStore-v1.js

```diff
@@ -26,7 +27,13 @@ async function getTransformationCodeV1(versionId) {
     const response = await fetchWithProxy(
       `${getTransformationURL}?versionId=${versionId}`
     );
-    if (response.status !== 200) {
+
+    if (response.status >= 500) {
```

Comment:
How about refactoring the response parsing logic to a separate function? It's repeated in multiple places.